RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -2021,3 +1979,94 @@ boolean prepare_food_xp_multi()
 	
 	return true;
 }
+
+void consumeStuff()
+{
+	if (auto_haveKramcoSausageOMatic())
+	{
+		auto_sausageWanted();
+	}
+
+	if (get_property("auto_limitConsume").to_boolean())
+	{
+		return;
+	}
+
+	if (bat_consumption())
+	{
+		return;
+	}
+	if (inAftercore())
+	{
+		return;
+	}
+	if(in_community())
+	{
+		cs_eat_spleen();
+		return;
+	}
+	if(in_kolhs())
+	{
+		kolhs_consume();
+		return;
+	}
+	if(in_robot())
+	{
+		robot_get_adv();
+		return;
+	}
+
+	// fills up spleen for Ed.
+	if (ed_eatStuff())
+	{
+		return;
+	}
+
+	boolean edSpleenCheck = (isActuallyEd() && my_level() < 11 && spleen_left() > 0); // Ed should fill spleen first
+	
+	if (my_adventures() < 10 && fullness_left() > 0 && is_boris())
+	{
+		borisDemandSandwich(true);
+	}
+
+	// guilty sprouts provide big stats
+	if(auto_is_valid($item[guilty sprout]) && canEat($item[guilty sprout]) && my_level() < 13 && !in_tcrs())
+	{
+		// attempt to eat spaghetti breakfast as can only be eaten as the first food of the day
+		if(my_level() >= 11 && my_fullness() == 0 && !get_property("_spaghettiBreakfastEaten").to_boolean())
+		{
+			autoEat(1, $item[Spaghetti Breakfast]);
+		}
+		
+		// important for leveling. Attempt to pull if we don't have one
+		pullXWhenHaveY($item[guilty sprout], 1, 0);
+
+		// use food to level if ready for it
+		if(prepare_food_xp_multi())
+		{
+			autoEat(1, $item[guilty sprout]);
+		}
+	}
```

Comment:
I would take this out. It's unnecessary in current Standard with the trainset & thus actively detrimental.
For people/paths without trainset yes it might be useful but it's a shotgun approach right now. Refactoring it for those situations where hitting level 14+ day 1 isn't trivial is more work than simply taking it out so it's your call but personally I would just remove it.

Reply:
Ya fair point. What about adding another condition for if we are power leveling? Then we know we are leveling slowly. Would require trying to consume something while power leveling which may not happen, so maybe this would go somewhere else too... I'll think about it

Reply:
Maybe add a day count check to the first conditional, like my_daycount() >=2? That way if someone is still less than level 13 (or maybe 11 or 12 to really emphasize it is really a stat gainer) and on day 2+ they would eat the sprout?

Reply:
Alrighty decided just adding a check for if we are power leveling is sufficient. This should be checked every loop of doTasks. May not work perfectly but believe it will work better now. Also moved the pull so only pull if we are actually going to eat it to resolve #1258